components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml

```diff
@@ -428,6 +428,20 @@
 <config_frazil_maximum_freezing_temperature>0.0</config_frazil_maximum_freezing_temperature>
 <config_frazil_use_surface_pressure>.false.</config_frazil_use_surface_pressure>
 
+<!-- land_ice -->
+<config_land_ice_draft_mode>'pressure-dependent'</config_land_ice_draft_mode>
+<config_land_ice_draft_mode ocn_grid="oQU240wLI">'data'</config_land_ice_draft_mode>
+<config_land_ice_draft_mode ocn_grid="ECwISC30to60E1r2">'data'</config_land_ice_draft_mode>
+<config_land_ice_draft_mode ocn_grid="SOwISC12to60E2r4">'data'</config_land_ice_draft_mode>
+<config_land_ice_draft_mode ocn_grid="ECwISC30to60E2r1">'data'</config_land_ice_draft_mode>
+<config_land_ice_draft_mode ocn_grid="IcoswISC30E3r5">'data'</config_land_ice_draft_mode>
+<config_land_ice_draft_mode ocn_grid="FRISwISC08to60E3r1">'data'</config_land_ice_draft_mode>
+<config_land_ice_draft_mode ocn_grid="FRISwISC04to60E3r1">'data'</config_land_ice_draft_mode>
+<config_land_ice_draft_mode ocn_grid="FRISwISC02to60E3r1">'data'</config_land_ice_draft_mode>
+<config_land_ice_draft_mode ocn_grid="FRISwISC01to60E3r1">'data'</config_land_ice_draft_mode>
+<config_land_ice_draft_mode ocn_grid="RRSwISC6to18E3r5">'data'</config_land_ice_draft_mode>
+<config_land_ice_rho_ocean>1028</config_land_ice_rho_ocean>
+
```

Comment:
> @cbegeman -- is this still NCC? For all configurations?

@jonbob Do you still have questions about this? It is NCC for all configurations **NOT** listed above as `config_land_ice_draft_mode='data'`. When `config_land_ice_draft_mode='data'`, it is BFB.

Reply:
@jonbob, correct me if this is not right, but I think what you were looking for is an update to the description of the PR that clarifies for which meshes (thus campaigns) this is NCC and for which it is BFB.  This will help folks to know whether this affects them.  If it's just labeled as NCC, it requires folks to dig pretty deep here to find out whether they're affected.

Reply:
Thanks @xylar -- yes, that's exactly what I was looking for